addons/l10n_ec/data/l10n_latam.document.type.csv

```diff
@@ -1,37 +1,38 @@
-"ec_dt_42",20,"42","Documento retención presuntiva y retención emitida por propio vendedor o por intermediario","Documento retención presuntiva y retención emitida por propio vendedor o por intermediario","invoice","DOC","base.ec",1,0
-"ec_dt_43",21,"43","Liquidacion para Explotacion y Exploracion de Hidrocarburos","Liquidacion para Explotacion y Exploracion de Hidrocarburos","invoice","DOC","base.ec",1,0
-"ec_dt_44",22,"44","Comprobante de Contribuciones y Aportes","Comprobante de Contribuciones y Aportes","invoice","DOC","base.ec",1,0
-"ec_dt_45",23,"45","Liquidación de medicina prepagada","Liquidación de medicina prepagada","purchase_liquidation","DOC","base.ec",1,0
-"ec_dt_47",24,"47","N/C por Reembolso Emitida por Intermediario","N/C por Reembolso Emitida por Intermediario","credit_note","NCR","base.ec",1,1
-"ec_dt_48",25,"48","N/D por Reembolso Emitida por Intermediario","N/D por Reembolso Emitida por Intermediario","debit_note","NDR","base.ec",1,1
-"ec_dt_49",26,"49","Proveedor Directo de Exportador Bajo Régimen Especial","Proveedor Directo de Exportador Bajo Régimen Especial","invoice","DOC","base.ec",1,0
-"ec_dt_50",27,"50","A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","invoice","DOC","base.ec",1,0
-"ec_dt_51",28,"51","N/C A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","N/C A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","credit_note","DOC","base.ec",1,1
-"ec_dt_52",29,"52","N/D A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","N/D A Inst. Estado y Empr. Públicas que percibe ingreso exento de Imp. Renta","debit_note","DOC","base.ec",1,1
-"ec_dt_294",30,"294","Liquidación de compra de Bienes Muebles Usados","Liquidación de compra de Bienes Muebles Usados","purchase_liquidation","LDC","base.ec",1,0
-"ec_dt_344",31,"344","Liquidación de compra de vehículos usados","Liquidación de compra de vehículos usados","purchase_liquidation","LDC","base.ec",1,0
-"ec_dt_364",32,"364","Acta Entrega-Recepción PET","Acta Entrega-Recepción PET","invoice","RPET","base.ec",1,0
-"ec_dt_370",33,"370","Factura operadora transporte / socio","Factura operadora transporte / socio","invoice","DOC","base.ec",1,0
-"ec_dt_371",34,"371","Comprobante socio a operadora de transporte","Comprobante socio a operadora de transporte","invoice","DOC","base.ec",1,0
-"ec_dt_372",35,"372","Nota de  crédito  operadora transporte / socio","Nota de  crédito  operadora transporte / socio","credit_note","DOC","base.ec",1,0
-"ec_dt_373",36,"373","Nota de  débito  operadora transporte / socio","Nota de  débito  operadora transporte / socio","debit_note","DOC","base.ec",1,0
+id,sequence,code,name,report_name,internal_type,doc_code_prefix,country_id/id,active,l10n_ec_check_format
```

Comment:
@vbe-odoo I used meld to compare the previous file and the new one and there are just a few specific changes. I can show you in the next meeting. This highlighting is wrong and has something to do with GitHub probably. I think we could close this comment.

Reply:
@jco-odoo here there's a GitHub problem as not everything was changed, only a few parts of every line, but GitHub was not smart enough to only highlight the changes.
This will be closed.

Reply:
There is like a change on every line almost, so it makes sense the diff is like that.  I was afraid that it would be an encoding problem.